UPGRADE-5.1.md

```diff
@@ -82,6 +82,11 @@ Notifier
  * [BC BREAK] The `EmailMessage::fromNotification()` and `SmsMessage::fromNotification()`
    methods' `$transport` argument was removed.
 
+OptionsResolver
+---------------
+
+ * The signature of method `OptionsResolver::setDeprecated()` has been updated to `OptionsResolver::setDeprecation(string $option, string $package, string $version, $message)`.
```

Comment:
this should be duplicated in ` UPGRADE-6.0.md`

Reply:
Also it looks like there is a typo: `setDeprecation` not exists at all.

Reply:
nice catch @Koc